pyServer/manifests/linux/aks

```diff
@@ -20,6 +20,9 @@ copy,/var/run/azure-vnet*
 copy,/run/azure-vnet*
 copy,/etc/cni/net.d/10-azure.conflist
 
+echo,### NPM logs ###
+copy,/var/log/azure-npm.log
```

Comment:
Can this logs contain any secrets? If it can't then let's add "noscan" for this one.

Reply:
added noscan since there are no secrets in these logs